installer/IF20_nvd_feed.sh

```diff
@@ -58,7 +58,7 @@ IF20_nvd_feed() {
           git clone --depth 1 -b main https://github.com/EMBA-support-repos/EPSS-data.git external/EPSS-data || ( echo "Could not install EMBA component NIST EPSS data feed" && exit 1 )
         fi
 
-        if [[ $(grep -l -E "cpe.*busybox:" external/nvd-json-data-feeds/* -r 2>/dev/null | wc -l) -gt 18 ]]; then
+        if [[ $(rg -l "cpe.*busybox:" external/nvd-json-data-feeds/* 2>/dev/null | wc -l) -gt 18 ]]; then
```

Comment:
I would not recommend using rg in the installer as we need to install it on the host first